<?xml version="1.0" encoding="UTF-8" ?>
<testsuite name="tst_Exception" timestamp="@TEST_START_TIME@" hostname="@HOSTNAME@" tests="2" failures="1" errors="0" skipped="0" time="@TEST_DURATION@">
  <properties>
    <property name="QTestVersion" value="@INSERT_QT_VERSION_HERE@"/>
    <property name="QtVersion" value="@INSERT_QT_VERSION_HERE@"/>
    <property name="QtBuild" value=""/>
  </properties>
  <testcase name="initTestCase" classname="tst_Exception" time="@TEST_DURATION@"/>
  <testcase name="throwException" classname="tst_Exception" time="@TEST_DURATION@">
    <failure type="fail" message="Caught unhandled exception"/>
  </testcase>
</testsuite>
